### ISSUE TITLE
TZ Issues

### ISSUE THREAD

**Issue by tacgnol** [ctrlo/GADS#141]
Something bit a bit screwy with the timezones. Both client and server have correct date and time and both in BST timezone:

```
$ hostname
www
$ date
Sun Apr  5 16:10:16 BST 2020
$ 

dhall@xenon ~                                                              [16:10:09] 
> $ date                                                                             
Sun  5 Apr 2020 16:10:13 BST
                                                                                      
dhall@xenon ~                                                              [16:10:13] 
> $         
```                               

If I import records, metadata shows them with a time that is off by an hour:

![Screenshot 2020-04-05 at 16 02 51](https://user-images.githubusercontent.com/4366890/78502055-f1739c80-7756-11ea-8222-76426a14dc2b.png)

Version history is correct:
|
![Screenshot 2020-04-05 at 16 03 22](https://user-images.githubusercontent.com/4366890/78502100-3566a180-7757-11ea-9e40-756f169fab8a.png)

If I edit a record, the meta data now changes to have an a created time of when I made the change, and the last edited time stays the same:

![Screenshot 2020-04-05 at 16 07 16](https://user-images.githubusercontent.com/4366890/78502168-8b3b4980-7757-11ea-80f6-3902e3ad5fa8.png)

Version history now shows two times. The initial date/time when the record was created, which appears to be in UTC, and then my latest update which is has the time in BST.

![Screenshot 2020-04-05 at 16 07 47](https://user-images.githubusercontent.com/4366890/78502198-bd4cab80-7757-11ea-9ccf-e6e68e5b3aeb.png)

As a result of this, I don't see updates for recently added records in the default view as the latest data appears to be older the original imported data.

I don't know whether I'm missing something in a config file somewhere?

**Comment by abeverley** [comment]
Thanks for the comprehensive bug report. This seems to have been a regression introduced in b77c0d4. Probably the best thing to do for the moment is to roll back to the commit just before that one (4b11a7b) and we'll work on a fix in the meantime.

**Comment by abeverley** [comment]
This has been fixed in 146bb73. A script to update any incorrect records was included in 91b55df.
